src/classes/AudioResponse.ts

```diff
@@ -1,6 +1,6 @@
 import UseCaseResponse from './UseCaseResponse';
 
-export class AudioResponse extends UseCaseResponse {
```

Comment:
Although not in Use. Why did you get rid of the Inheritance ?

Reply:
Ein `UseCaseResponse` wird ja nie selbst instanziiert, sondern nur seine Unterklassen. Man könnte `UseCaseResponse`also in eine abstrakte Klasse machen, aber da es keine Funktionalität selbst enthält, habe ich mich für ein Interface entschieden.